debian/control

```diff
@@ -8,7 +8,7 @@ Standards-Version: 3.8.3
 Package: tron
 Architecture: all
 Homepage: http://github.com/yelp/Tron
-Depends: python2.7, ${shlibs:Depends}, ${misc:Depends}
+Depends: python2.7, libyaml-dev, ${shlibs:Depends}, ${misc:Depends}
```

Comment:
why does it need libyaml-dev? This should only be a build dep

Reply:
good point, i got confused what is installed when